src/components/expanded-state/swap-details/SwapDetailsContractRow.js

```diff
@@ -160,17 +160,21 @@ export default function SwapDetailsContractRow({
     );
   }, [asset, handleCopyContractAddress]);
 
+  const [menuVisible, setMenuVisible] = useState(false);
```

Comment:
You forgot to import `useState`

Reply:
Would probably work anyway since it will likely be somewhere in bundle, but it should be there anyway

Reply:
it works, because we have have it set in global scope
![image](https://user-images.githubusercontent.com/25709300/178492268-b012adb8-6c32-4847-ba5c-78a139b1780a.png)

At some point i was thinking this is smart